tests/dummy/app/templates/chips.hbs

```diff
@@ -0,0 +1,46 @@
+{{#paper-toolbar}}
+  <h2 class="md-toolbar-tools">
+    {{#paper-sidenav-toggle class="menu-sidenav-toggle" }}
+      {{paper-icon icon="menu"}}
+    {{/paper-sidenav-toggle}}
+    <span>Chips</span>
+  </h2>
+{{/paper-toolbar}}
+{{#paper-content classNames="md-padding demo-buttons" }}
+  <div class="doc-content">
+    {{#paper-card}}
+      <md-card-content>
+        <h2>Default Template</h2>
+        {{paper-chips content=fruitNames readOnly=readOnly removeItem="removeItem" addItem="addItem" placeholder="Add a tag"}}
+
+        <h2>Custom Template</h2>
+        {{#paper-chips readOnly=readOnly removeItem="removeCustomItem" addItem="addCustomItem" placeholder="Add a tag" content=customFruitNames as |item|}}
+          <strong>{{item}}</strong>
+          <em>(fruit)</em>
+        {{/paper-chips}}
+
+        <h2>Contact Chips</h2>
+        {{paper-contact-chips class="md-contact-chips" readOnly=readOnly removeItem="removeContact" addItem="addContact" placeholder="Add a tag" content=contacts source=remainingContacts}}
+
+        {{#paper-checkbox value=readOnly onChange=(action (mut readOnly))}}Read Only{{/paper-checkbox}}
+      </md-card-content>
+    {{/paper-card}}
+
+    <h3>Template</h3>
+    {{#code-block language='handlebars' }}
+&lt;h2&gt;Default Template&lt;/h2&gt;
+\{{paper-chips content=fruitNames readOnly=readOnly removeItem=&quot;removeItem&quot; addItem=&quot;addItem&quot; placeholder=&quot;Add a tag&quot;}}
```

Comment:
These are really long. Maybe wordwrap? Contributing has, I think, the indentation style for this, but the `{{` and `}}` should be at the same indentation level.